src/components/Select/Select.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import classnames from 'classnames';
+
+import { px, useNormalizedInputProps } from '../../utils';
+
+import { InputProps } from '../Input/Input';
+
+export interface SelectProps extends InputProps {
+  /**
+   * Option elements that are selectable
+   */
+  children: React.ReactNode;
+
+}
+
+const Select = React.forwardRef((
```

Comment:
import {forwardRef} from 'react'

Reply:
It is a convention. I prefer verbosity to explicitly show what is coming from  React. It also does not require me to continuously update my import statement if and when I need to pull another method from React.